operator/internal/manifests/openshift/internal/dashboards/build.go

```diff
@@ -0,0 +1,61 @@
+package dashboards
+
+import (
+	"embed"
+	"fmt"
+	"io/fs"
+	"path"
+)
+
+const (
+	staticDir                       = "static"
+	lokiStackChunkDashboardFile     = "grafana-dashboard-lokistack-chunks.json"
+	lokiStackReadsDashboardFile     = "grafana-dashboard-lokistack-reads.json"
+	lokiStackWritesDashboardFile    = "grafana-dashboard-lokistack-writes.json"
+	lokiStackRetentionDashboardFile = "grafana-dashboard-lokistack-retention.json"
```

Comment:
My intention is to prevent alternating names between releases to not worry on how Grafana will provision things, i.e. depending on Grafana/OCP's-Grafana to delete old dashboards and create new one based on ConfigMap data key changes.

Reply:
Fair point. Does it make sense to move them into the `_test` file then and maybe make it a `expectedDashboardFiles` slice reducing the repeated testing code a bit?

edit: Aren't the number of files and their names fixed by the Jsonnet code in this repo? (`dashboards.jsonnet`)